packages/boxel/addon/components/boxel/control-panel/index.gts

```diff
@@ -0,0 +1,71 @@
+import Component from '@glimmer/component';
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+//@ts-expect-error glint does not think this is consumed-but it is consumed in the template https://github.com/typed-ember/glint/issues/374
+import { hash } from '@ember/helper';
+import cn from '@cardstack/boxel/helpers/cn';
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+
+import '@cardstack/boxel/styles/global.css';
+import './index.css';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+  };
+  Blocks: {
+    default: [{ Item: typeof Item }],
+  };
+}
+
+interface ItemSignature {
+  Element: HTMLElement
+  Args: {
+    icon: string;
+    isActive: boolean;
+    title: string;
+  },
+  Blocks: {
+    default: [],
+  }
+}
+
+const Item: TemplateOnlyComponent<ItemSignature> =
+<template>
+  <section class="boxel-control-panel__item">
+    <heading
+      class={{
+        cn
+        "boxel-control-panel__item-heading"
+        boxel-control-panel__item-heading--is-active=@isActive
+      }}>
+      {{svgJar
+        @icon
+        width="18px"
+        height="18px"
+        class="boxel-control-panel__item-icon"
+      }}
+      {{@title}}
+    </heading>
+    <div class="boxel-control-panel__item-body">
+      {{yield}}
+    </div>
+  </section>
+</template>;
+
+export default class ControlPanel extends Component<Signature> {
+  get jortle(): number {
+    return 3;
+  }
+
+  <template>
+    <nav class="boxel-control-panel">
```

Comment:
I don't think we can guarantee that this is used as a `nav`. There are various screens that look like they offer actions, not navigation.

Reply:
ya, changed now, thanks